components/eamxx/src/dynamics/homme/eamxx_homme_iop.cpp

```
@@ -594,9 +594,9 @@ apply_iop_forcing(const Real dt)
             // Restrict nudging of T and qv to certain levels if requested by user
             // IOP pressure variable is in unitis of [Pa], while iop_nudge_tq_low/high
             // is in units of [hPa], thus convert iop_nudge_tq_low/high
-            Mask nudge_level;
-            for (int p=0; p<Pack::n; ++p) {
-              const auto lev = k*Pack::n + p;
+            Mask nudge_level(false);
```

I assume I should default this mask to all false?